src/seedu/addressbook/AddressBook.java

```diff
@@ -218,11 +218,11 @@ public static void main(String[] args) {
      * ====================================================================
      */
     private static void showWelcomeMessage() {
-        showToUser(DIVIDER, DIVIDER, VERSION, MESSAGE_WELCOME, DIVIDER);
+        showMessageToUser(DIVIDER, DIVIDER, VERSION, MESSAGE_WELCOME, DIVIDER);
```

Comment:
i like your thought on making it explicit; but here it is not adding too much value.